File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/TransportersCreateVehiclesRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class TransportersCreateVehiclesRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("LicensePlateNumber")]
        public string? LicensePlateNumber { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Make")]
        public string? Make { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Model")]
        public string? Model { get; set; }
    }
}

File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/TransportersCreateVehiclesRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class TransportersCreateVehiclesRequest {
    @JsonProperty("LicensePlateNumber")
    public String licensePlateNumber;
    @JsonProperty("Make")
    public String make;
    @JsonProperty("Model")
    public String model;
}
